docs/OV_Runtime_UG/migration_ov_2_0/common_inference_pipeline.md

```diff
@@ -135,7 +153,25 @@ OpenVINO™ Runtime API 2.0:
 
 @snippet docs/snippets/ov_common.cpp ov_api_2_0:inference
 
-## 7. Process the Inference Results
+## 8. Start asynchronous inference and wait for result
```

Comment:
7-8 steps are all about inference. Maybe we can have a single inference step, but it will have selector between sync and async versions? 
- Sync: just call r.infer();
- Async: set callback + start_async + wait.

Reply:
added